src/dotnet-svcutil/lib/src/Bootstrapper/SvcutilBootstrapper.cs

```diff
@@ -0,0 +1,254 @@
+            {
+                // creating the directory prevents the bootstrapper project from deleting it as it doesn't own it, files will not be deleted either.
+                Directory.CreateDirectory(Path.GetDirectoryName(projectFullPath));
+            }
+
+            // If the target framework was provided, it was validated already when processing params.
+            // Bootstrapping is enabled only for netcoreapp or netstandard TFM, let's check.
+
+            bool isSupportedTFM = TargetFrameworkHelper.IsSupportedFramework(this.Options.TargetFramework.FullName, out var frameworkInfo);
+            Debug.Assert(frameworkInfo.IsDnx, "The bootstrapper has been enabled for a non-DNX platform!");
+
+            ToolConsole.WriteLineIf(ToolConsole.Verbosity >= Verbosity.Verbose, Resource.CreatingProjectMsg);
+
+            using (await SafeLogger.WriteStartOperationAsync(logger, $"Creating project file: \"{projectFullPath}\"").ConfigureAwait(false))
+            {
+                var svcutilPkgRef = ProjectDependency.FromPackage(Tool.AssemblyName, Tool.PackageVersion);
+
+                this.MSBuildProj = await MSBuildProj.DotNetNewAsync(projectFullPath, logger, cancellationToken).ConfigureAwait(false);
+                this.MSBuildProj.AddDependency(svcutilPkgRef);
+
+                // NOTE: If post v2.0 NetStandard ships a different version from NetCore the table below needs to be updated!
+                var targetFramework = frameworkInfo.FullName;
+                if (isSupportedTFM && frameworkInfo.IsKnownDnx)
+                {
+                    if (frameworkInfo.Name == FrameworkInfo.Netstandard)
+                    {
+                        targetFramework = FrameworkInfo.Netcoreapp + TargetFrameworkHelper.NetStandardToNetCoreVersionMap[frameworkInfo.Version];
+                    }
+                    this.MSBuildProj.TargetFramework = targetFramework;
+                }
+                // else
+                // The TFM is unknown: either, it was not provided or it is a version not yet known to the tool,
+                // we will use the default TF from the generated project.
+            }
+
+            foreach (ProjectDependency dependency in this.Options.References)
+            {
+                this.MSBuildProj.AddDependency(dependency);
+            }
+
+            if (!string.IsNullOrEmpty(this.Options.RuntimeIdentifier))
+            {
+                this.MSBuildProj.RuntimeIdentifier = this.Options.RuntimeIdentifier;
+            }
+
+            // Don't treat warnings as errors so the bootstrapper will succeed as often as possible.
+            this.MSBuildProj.ClearWarningsAsErrors();
+
+            await this.MSBuildProj.SaveAsync(logger, cancellationToken).ConfigureAwait(false);
+        }
+
+        private static readonly string ProgramClass =
+            "using System; " + Environment.NewLine +
+            "namespace SvcutilBootstrap {" + Environment.NewLine +
```

Comment:
I would prefer to see this using a literal string without the Environment.NewLine. I know that would cause the new line character to be a Windows newline but as the target is the c# compiler which is tolerant, I don't think it's necessary to make sure the correct platform newline character is used. Ideally this would be in a file included as an assembly resource but that might be over-engineering things.